.github/workflows/testbuild.yml

```diff
@@ -45,23 +45,15 @@ jobs:
           echo "Building for $CHANGED_LOCATIONS"
           echo "CHANGED_LOCATIONS=$CHANGED_LOCATIONS" >> "$GITHUB_ENV"
 
-      - if: ${{ github.event_name == 'pull_request' }}
-        name: Build changed locations
+      - name: Build changed locations
         run: |
           if [ -n "${CHANGED_HOSTS}" ] || [ -n "${CHANGED_LOCATIONS}" ] ; then
             ansible-playbook play.yml --limit "${CHANGED_HOSTS:+$CHANGED_HOSTS,}$CHANGED_LOCATIONS"
           fi
           mkdir -p ./tmp/images
 
-      - if: ${{ github.event_name == 'push' }}
-        name: Update wiki.freifunk.net
-        run: |
-          ansible-playbook play.yml --tags wiki --limit "${CHANGED_HOSTS:+$CHANGED_HOSTS,}$CHANGED_LOCATIONS"
-        continue-on-error: true
-
-      - if: ${{ github.event_name == 'pull_request' }}
-        name: Store build output
+      - name: Store build output
         uses: actions/upload-artifact@v1
         with:
-          name: ansibleimages
-          path: ./tmp/images
+          name: ansibleconfigs
+          path: /tmp/config
```

Comment:
that's not working...

Reply:
I know, was just a quick test.
>I dont have to much time right now to dig in to this, feel free to propose something. Otherwise i will take care when i have time again

Reply:
It annoyed me to have this PR open so i fixed it. In addition i renamed everything to account for the change of the task